### ISSUE TITLE
Wrong DAI/ARS price precision for bitso

### ISSUE THREAD

**Issue by aurelianito** [ccxt/ccxt#7925]
Using latest python library (installed with pip, version 1.37.18, as informed in `ccxt.__version__`) amount precision for bitso is `8` but the `tick_size` informed by bitso is `0.01`. If the precision mode for bitso is kept, I would expect it to be `2`

Steps to reproduce:

```
from ccxt import bitso

b = bitso({})
b.load_markets()
print(b.market('DAI/ARS')['precision']['price'])
print(b.market('DAI/ARS')['info']['tick_size'])
```

**Comment by aurelianito** [comment]
With the proper credentials if I use more than 2 decimal places it generates the following error:
`ExchangeError: bitso {"success":false,"error":{"code":"0407","message":"Incorrect price 170.111"}}`

**Comment by aurelianito** [comment]
Is there any extra info required to solve this? How can I help?

**Comment by kroitor** [comment]
@aurelianito checking this, will be back to you with the answer asap!

**Comment by aurelianito** [comment]
FYI, if I set `b.markets['DAI/ARS']['precision']['price']=2` then the bug is fixed for the `b` object.

**Comment by kroitor** [comment]
@aurelianito this issue is fixed if you update to CCXT version 1.37.23. Thx for reporting it! Please, note that we have changed the `exchange.precisionMode` to `TICK_SIZE`, so, Bitso's precision values are now floats. Please let us know if that solves the issue on your side or not.

**Comment by aurelianito** [comment]
It seems to solve the issue. Thanks! :+1: :smile:
